.coveragerc

```diff
@@ -1,8 +1,8 @@
 [run]
-source = project_name
+source = fastapi_project_template
```

Comment:
was this change because of the CI rename workflow?

maybe we need to add a way to prevent that workflow to run on forks

Reply:
@rochacbruno Looks like it. Please reject this PR.